snuba/query/processors/object_id_rate_limiter.py

```diff
@@ -37,20 +59,15 @@ def process_query(self, query: Query, request_settings: RequestSettings) -> None
             if ex.rate_limit_name == self.rate_limit_name:
                 return
 
-        obj_ids = get_object_ids_in_query_ast(query, self.object_column)
-        if not obj_ids:
-            return
-
-        # TODO: Add logic for multiple IDs
-        obj_id = obj_ids.pop()
-
         object_rate_limit, object_concurrent_limit = get_configs(
             [
                 (self.per_second_name, DEFAULT_LIMIT),
                 (self.concurrent_name, DEFAULT_LIMIT),
             ]
         )
-
+        obj_id = self.get_object_id(query, request_settings)
+        if obj_id is None:
+            return
```

Comment:
This way we are adding this RateLimitParameters class to the rate limitier for all project_id+referrer pairs, which is a huge amount. Each one of them will write a key in Redis. 
We do not need that. 
Let's return a rate limit object only if there is a valid config for that specific referrer and that specific project without adding a default. 
That will ignore this rule for the majority of projects and referrer